src/main/java/com/semmle/jira/addon/workflow/LgtmTransitionNotificationFunction.java

```diff
@@ -121,7 +117,7 @@ private Transition getTransitionParam(Map<?, ?> args) throws InvalidInputExcepti
       throw new InvalidInputException("Missing value for 'transition' parameter");
```

Comment:
I'm hitting this error when trying to suppress alerts through the Jira UI.

Reply:
(Haven't yet understood this PR sufficiently to work out why...)

Reply:
Works fine if I go to the function config page for each of the transitions and set the transition type manually. So I guess it's just not getting initialised properly for some reason? (This was with a fresh Jira instance, no migration from a previous plugin version...)

Reply:
I think the workflow.xml needed fixing. Could you try again?

Reply:
Still no luck, I'm afraid. Same error as before, and again goes away if I update the transition type for the post function manually.

Reply:
Also, the post function for the  `unsuppress` transition is being initialised with a transition of type of `suppress`, so I think they're both just on 'default' somehow.

Reply:
Oh now wait, I'm talking rubbish, that _has_ fixed it!